CapabilityAgent.js

```diff
@@ -4,8 +4,8 @@
 'use strict';
```

Comment:
Remove `use strict` as this is using ES6.

Reply:
Removed, thanks!